### DISCUSSION TITLE
Cutoff new meter value

### DISCUSSION THREAD

**Discussion by michalwu** [patrickvorgers/Home-Assistant-Import-Energy-Data#66] [Q&A] [answered]
Hello

This is my first try. All went well, but for some reason home assistant was behaving strange on energy tab. Looked like it was in pain ;) and no history data so I decided to start all over. I do not think I fully understand the sense behind one cutoff value, and there is a chance I blew this part.
Cutoff new meter - is it first value (from entity), that I have in NEW home assistant (new - the one I am importing to)?

mike

**Comment by patrickvorgers** [comment, marked as answer]
Hi,

Do you have some more information?
- Sensor you are trying to import to
- Source you are trying to import from
- Generated CSV file
- Settings you use in the import script

The "Cutoff new meter" value is used to detect a new energy meter. So imagine the situation where we have the following readings:
- 2300
- 2313
- 12
- 14
- 16

You see a drop in the reading, probably at that point a new energy meter was installed. This first value (12) has to be lower than the "Cutoff new meter" value to be accepted as a "new meter" case.

**Reply by michalwu** [reply]
About my setup - I will try again just to make sure I did all necessary steps the right way, then I will complain with full details.
For now my suspect is cutoff value.
So if I am not changing meters, but just trying to import history, then my cutoff new meter value should be way higher than last value I have in this sensor?

**Reply by michalwu** [reply]
I tried again. This time exported data directly from db. It was a lot easier. And it did the trick.
Thank you so much!

For ppl looking for answers. This is my part of import script. No meter change. Quite high invalid values.

/*                          name                                sensor_id correction cutoff_new_meter cutoff_invalid_value */
INSERT INTO SENSORS VALUES ('sensor_id_elec_feed_in_tariff_1',  97,        1.0,    0,            10000.0); /* Change */
INSERT INTO SENSORS VALUES ('sensor_id_elec_feed_out_tariff_1', 90,        1.0,    0,            10000.0); /* Change */
INSERT INTO SENSORS VALUES ('sensor_id_elec_solar',             75,      1.0,    0,            10000.0); /* Change */
/* FOR NORMAL USAGE THERE SHOULD BE NO CHANGES NEEDED AFTER THIS POINT */
